lib/money_accessor.rb

```diff
@@ -0,0 +1,27 @@
+module MoneyAccessor
+  def self.included(base)
+    base.extend(ClassMethods)
+  end
+
+  module ClassMethods
+    def money_accessor(*columns)
+      [columns].flatten.each do |name|
+        define_method(name) do
+          value = instance_variable_get("@#{name}")
+          value.blank? ? nil : Money.new(value)
+        end
+
+        define_method("#{name}=") do |value|
+          if value.blank? || !value.respond_to?(:to_money)
```

Comment:
I don't like that it's just discarding it, but I guess the point is to have the exact same behaviour as `ActiveRecordHooks`.